doc/cephfs/troubleshooting.rst

```diff
@@ -5,21 +5,33 @@
 Slow/stuck operations
 =====================
 
-If you are experiencing apparent hung operations, the first task is to identify
-where the problem is occurring: in the client, the MDS, or the network connecting
-them. Start by looking to see if either side has stuck operations
-(:ref:`slow_requests`, below), and narrow it down from there.
+Sometimes CephFS operations hang. The first step in troubleshooting them is to
+locate the problem causing the operations to hang. Problems present in three
+places:
 
-We can get hints about what's going on by dumping the MDS cache ::
+#. in the client
+#. in the MDS
+#. in the network that connects the client to the MDS
 
-  ceph daemon mds.<name> dump cache /tmp/dump.txt
+First, use the procedure in :ref:`slow_requests` to determine if the client has
+stuck operations or the MDS has stuck operations.
 
-.. note:: The file `dump.txt` is on the machine executing the MDS and for systemd
-	  controlled MDS services, this is in a tmpfs in the MDS container.
-	  Use `nsenter(1)` to locate `dump.txt` or specify another system-wide path.
+Dump the MDS cache. The contents of the MDS cache will be used to diagnose the
+nature of the problem. Run the following command to dump the MDS cache:
 
-If high logging levels are set on the MDS, that will almost certainly hold the
-information we need to diagnose and solve the issue.
+.. prompt:: bash #
+
+   ceph daemon mds.<name> dump cache /tmp/dump.txt
+
+.. note:: MDS services that are not controlled by systemd, dump the file file
```

Comment:
"file file"

Remove comma after systemd

But with modern releases I should think that the MDS is always managed by systemd so we wouldn't have to describe a scenario where it isn't.

Reply:
I'm going to go ahead and merge this and make a note to myself that we have unfinished business here. Then we can raise a PR to deal specifically with this is-it-or-isn't-it managed by systemd issue.

Also, in the near future: some reshuffling of `troubleshooting.rst` so that it walks a reader through a unified procedure instead of sending the reader through a series of hyperlinks.